include/fastdds/rtps/participant/RTPSParticipant.h

```diff
@@ -259,6 +260,28 @@ class RTPS_DllAPI RTPSParticipant
 
 #endif // if HAVE_SECURITY
 
+#ifdef FASTDDS_STATISTICS
+
+    /*
+     * Adds a listener to receive statistics backend callbacks
+     * @param listener
+     * @param kind mask that specifies which callbacks to receive
+     */
```

Comment:
There is no return information. It would be interesting to include it having as a base the test cases done in #1827. Please, include this information in all the methods declared in this PR.